app/models/collection.rb

```diff
@@ -0,0 +1,8 @@
+# typed: strict
+class Collection < ApplicationRecord
+  has_many :items
+
+  # Define the attributes with their respective data types and constraints
+  validates :name, presence: true
+  validates :description, presence: true
+end
```

Comment:
Consider adding a unique constraint to the 'name' attribute to ensure that each collection has a unique name. This will help maintain data integrity.